tensorflow/lite/tools/cmake/modules/gemmlowp.cmake

```diff
@@ -40,6 +40,7 @@ endif()
 
 set(GEMMLOWP_SOURCE_DIR "${gemmlowp_SOURCE_DIR}" CACHE PATH "Source directory")
 add_subdirectory(
```

Comment:
Our internal CI fails here with

```
CMake Error at tools/cmake/modules/gemmlowp.cmake:42 (add_subdirectory):
  add_subdirectory called with incorrect number of arguments
Call Stack (most recent call first):
  tools/cmake/modules/Findgemmlowp.cmake:18 (include)
  CMakeLists.txt:151 (find_package)
```

I'm not sure what you are trying to achieve as the [`add_subdirectory` doc](https://cmake.org/cmake/help/latest/command/add_subdirectory.html) do indeed specify that there is only one source directory and one binary directory taken as a parameter.

Reply:
You are right, I was trying different approaches. In the end I messed up the commit. Should be fixed now.